src/components/toasts/OfflineToast.js

```diff
@@ -4,7 +4,7 @@ import Toast from './Toast';
 
 const OfflineToast = () => {
   const isConnected = useInternetStatus();
-  return <Toast icon="offline" isVisible={isConnected} text="Offline" />;
+  return <Toast icon="offline" isVisible={!isConnected} text="Offline" />;
```

Comment:
😂 it was working before! but i reversed the logic inside Toast.js so i needed to update it here too

reason for logic change:
the previous logic would cause the Toast to immediately show itself and then hide, if its default “isVisible” value was false